lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -259,6 +260,39 @@ defmodule ExUnit.Runner do
     end
   end
 
+  defp maybe_create_tmp_dir(test, context, %{tmp_dir: true}) do
+    create_tmp_dir!(test, "", context)
+  end
+
+  defp maybe_create_tmp_dir(test, context, %{tmp_dir: path}) when is_binary(path) do
+    create_tmp_dir!(test, path, context)
+  end
+
+  defp maybe_create_tmp_dir(_, _, %{tmp_dir: other}) do
```

Comment:
Should it maybe accept `tmp_dir: false` too just in case someone wants to override a `@moduletag`?

Reply:
Good call, updated!